python/eos/figure/plot.py

```diff
@@ -14,143 +14,255 @@
 # this program; if not, write to the Free Software Foundation, Inc., 59 Temple
 # Place, Suite 330, Boston, MA  02111-1307  USA
 
-import eos
-from .item import ItemFactory
-
+from abc import ABC, abstractmethod
 from collections import OrderedDict
```

Comment:
Ditto re `OrderedDict` (this line of code may be old enough that is wasn't true when it got written at least)

Reply:
Resolved.